packages/sanity/src/core/store/bundles/useBundleOperations.ts

```diff
@@ -0,0 +1,50 @@
+import {uuid} from '@sanity/uuid'
+import {useCallback} from 'react'
+
+import {useAddonDataset} from '../../studio/addonDataset/useAddonDataset'
+import {type BundleDocument} from './types'
+
+// WIP - Raw implementation for initial testing purposes
+export function useBundleOperations() {
+  const {client} = useAddonDataset()
```

Comment:
Nice! 🙂 I didn't know `useAddonDataset` existed.

Reply:
It was initially part of comments and then moved to core when we built tasks.